app/scripts/services/os-admin/index.js

```diff
@@ -91,7 +91,9 @@ function getDataPackages(authToken, userid) {
   if (authToken) {
     url += '&jwt=' + encodeURIComponent(authToken);
   }
-  url += '&package.owner=' + encodeURIComponent(JSON.stringify(userid));
+  if (userid) {
```

Comment:
Okay, just wanted to make sure that this is basically a sanity check and not some temporal state that we expect to happen.